Update support-for-keeping-add-ins-enabled.md (#1634)
MicrosoftDocs/VBA-Docs: outlook/Concepts/Getting-Started/support-for-keeping-add-ins-enabled.md

|Values|Specify the value as follows:<br>0 = always disabled (blocked)<br>1 = always enabled<br>2 = configurable by the user and not blocked by the **Block all unmanaged add-ins** policy setting when enabled.|

> [!NOTE]

> > -To obtain the ProgID for an add-in, use the Windows Registry Editor on the client computer where the add-in is installed. Locate and copy the registry key names as it is respecting capital and lower-case (Case sensitive) under : HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Office\<application>\Addins or HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\<application>\Addins. 

Reviewer: ```suggestion
> To obtain the ProgID for an add-in, use the Windows Registry Editor on the client computer where the add-in is installed. Copy the registry key names under found: HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Office\<application>\Addins or HKEY_CURRENT_USER\SOFTWARE\Microsoft\Office\<application>\Addins. Registry key names are **case sensitive**.
```
